config/manifests/probesPatch.yaml

```diff
@@ -0,0 +1,48 @@
+- op: add
+  path: /spec/customresourcedefinitions/owned/0/specDescriptors/-
+  value:
+    description: Hide liveness probe's Exec field
+    displayName: Livness Probe's Exec
+    path: probes.liveness.exec
+    x-descriptors:
+      - urn:alm:descriptor:com.tectonic.ui:hidden
+- op: add
+  path: /spec/customresourcedefinitions/owned/0/specDescriptors/-
+  value:
+    description: Hide liveness probe's TCP Socket field
+    displayName: Livness Probe's Exec
```

Comment:
Looks like this display name is wrong

Reply:
This was taken directly from WLO.  From Leo: "probes’s fields under exec are hidden, so display name doesn’t matter, since it won’t render"